libs/langchain-community/src/graphs/neo4j_graph.ts

```diff
@@ -1,4 +1,6 @@
 import neo4j, { RoutingControl } from "neo4j-driver";
+import { createHash } from "crypto";
```

Comment:
I am not sure if this is intended to work in web environments (e.g. Next.js) but I would suggest using this:

https://github.com/langchain-ai/langchainjs/blob/09db56a3521fd96db33a2deaf8378b2ab29cbb57/langchain/src/embeddings/cache_backed.ts#L1